topsort-api-v2.yml

```diff
@@ -111,20 +79,21 @@ paths:
                   items:
                     oneOf:
```

Comment:
Please define this as a type, otherwise when generating code it end ups with a made up name. For example for the axios generator ends up with the name `InlineObject`. Same for the response.